draft-ietf-quic-transport.md

```diff
@@ -451,11 +467,9 @@ following sections.
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+
-|0|C|K|1|0|T T T|
+|0|K|1|1|0|T T T|
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                                                               |
-+                     [Connection ID (64)]                      +
-|                                                               |
+|                Destination Connection ID (0..148)           ...
```

Comment:
Maybe this needs be corrected to `Destination Connection ID (0/32..144)`?

I see the same issue in the definition of the Stateless Reset (it says 148) and also in short header definition in the invariants draft.